extensions/ci/main.ts

```diff
@@ -0,0 +1,44 @@
+import { PhylumApi } from "phylum";
+
+// Ensure required arguments are present.
+if (Deno.args.length < 4 || Deno.args.length > 5) {
+  console.error(
+    "Usage: phylum ci <PROJECT> [--group <GROUP>] <LABEL> <BASE> <LOCKFILE>",
```

Comment:
The `phylum analyze` command, and the [current implementation of `phylum-ci`](https://github.com/phylum-dev/phylum-ci/blob/main/src/phylum/ci/ci_base.py#L411-L445), allows for multiple lockfiles to be specified. Can this (sample) extension be modified to also allow for multiple lockfiles? Javascript/Typescript is not something I'm terribly familiar with.

Reply:
Should be trivial, yeah.